libs/vkd3d/command.c

```diff
@@ -8372,6 +8420,21 @@ HRESULT d3d12_command_list_create(struct d3d12_device *device,
     return S_OK;
 }
 
+static void STDMETHODCALLTYPE d3d12_command_queue_set_name(struct d3d12_command_queue *command_queue, const char *name)
+{
+    VkQueue vk_queue;
+
+    if (!(vk_queue = vkd3d_queue_acquire(command_queue->vkd3d_queue)))
```

Comment:
I guess we need to lock here since app doesn't expect different ID3D12Queues to resolve to same VkQueue?

Reply:
I'd argue that we shouldn't really name the Vulkan queue then, especially since every command queue is currently guaranteed to point to the same VkQueue.

Reply:
Agree

Reply:
Made this just do nothing then.